app/controllers/articles_controller.rb

```diff
@@ -3,6 +3,7 @@
 # Controller for articles model
 class ArticlesController < ApplicationController
   # http_basic_authenticate_with name: "dhh", password: "secret", except: [:index, :show]
+  # before_action :authenticate_user!, except: [:index, :show]
```

Comment:
I think we need this line, to protect some actions for the article controller(for example show), what do you think?

Reply:
I think it depends on the purpose. There, I just hide the add, comment and delete buttons for non-logged users.